Multisig/Logic/Models/Safe+CoreDataProperties.swift

```diff
@@ -20,4 +20,5 @@ extension Safe {
     @NSManaged public var name: String?
     @NSManaged public var selection: Selection?
     @NSManaged public var network: Network?
+    @NSManaged public var contractVersion: String?
```

Comment:
is this a breaking change for the database?

Reply:
this is an additional field with default empty string value